tests/test_kernel32/test_process.py

```diff
@@ -240,3 +240,14 @@ def test_terminates_process(self):
         TerminateProcess(handle, 42)
         process.communicate()
         self.assertEqual(process.returncode, 42)
+
+
+class TestCreateToolhelp32Snapshot(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.CreateToolhelp32Snapshot`
+    """
```

Comment:
You can do something like this:

```python
with self.assertRaises(WindowsAPIError):
    CreateToolhelp32Snapshot(library.TH32CS_SNAPPROCESS, <invalid process id>)
```

Where `<invalid process id>` is either a process you start and kill with `subprocess.check_call()` or a process id which is completely invalid (any number not 0, 4 or divisible by 4)